Multisig/UI/WalletConnectV2/WalletConnectManager.swift

```diff
@@ -183,15 +199,98 @@ class WalletConnectManager {
         precondition(Thread.isMainThread)
         Safe.removeSession(topic: topic)
     }
-    
+
     func getSessions(topics: [String]) -> [Session] {
-        Sign.instance.getSessions().filter({topics.contains($0.topic)})
+        Sign.instance.getSessions().filter({ topics.contains($0.topic) })
     }
+
+    private func handle(request: Request) {
+        dispatchPrecondition(condition: .notOnQueue(.main))
+
+        guard let session = getSessions(topics: [request.topic]).first else {
+            reject(request: request)
+            return
+        }
+
+
+        guard let safe = Safe.by(topic: request.topic) else {
+            reject(request: request)
+            return
+        }
+
+        if request.method == "eth_sendTransaction" {
+
+            // make transformation of incoming request into internal data types
+            // and fetch information about safe from the request
+
+            guard let safeInfo = try? App.shared.clientGatewayService.syncSafeInfo(
+                    safeAddress: safe.addressValue, chainId: safe.chain!.id!)
```

Comment:
Wrong Indentation

Reply:
It is now indented as it was in your commit. If that's not correct, then I need to know how it should be indented.